### DISCUSSION TITLE
Not listing all duplicate photos

### DISCUSSION THREAD

**Discussion by Alister-Zeus** [hsoft/dupeguru#1006] [Q&A]
I scan a folder of ~22,000 images spread though out a number of sub levels

If I set say the similarity to 65 and scan I get a number of duplicates. I keep scanning at the 65 setting till there are none.

Then I change the value to 64 - suddenly I'm getting matches with value of 76, 84, 67 etc appearing.

Is this normal?

**Comment by arsenetar** [comment]
I believe this is possible due to how the similarity percentage and the related image block comparison code works. So it would be possible to see "higher" percentage matches after lowering the similarity percentage.  There may be a way to prevent this from happening, but I would have to look into it more, and I believe it would sacrifice comparison speed to accomplish it.
